modin/numpy/arr.py

```diff
@@ -490,10 +490,10 @@ def max(
                         f"operand was set up as a reduction along axis 1, but the length of the axis is {out.shape[0]} (it has to be 1)"
                     )
                 if initial is not None and out is not None:
-                    out._update_inplace(
-                        (numpy.ones_like(out) * initial)._query_compiler
-                    )
-                if where is not False or out is not None:
+                    out._update_inplace = (
+                        numpy.ones_like(out) * initial
+                    )._query_compiler
```

Comment:
uhh, `out._update_inplace =` feels awfully wrong

Reply:
Yep - that's definitely wrong!